soccer/gameplay/tactics/defense.py

```diff
@@ -71,6 +113,21 @@ def execute_running(self):
             # raise RuntimeError("Defense tactic requires a goalie id to be set")
 
             # TODO: move a lot of this code into modules in the evaluation folder
+
+            #main.system_state().draw_circle(robocup.Point(0, 0), constants.Field.ArcRadius * 2,constants.Colors.Red, "Clear Ball")
+
+    def execute_clearing(self):
```

Comment:
For some reason if it isn't called continuously it doesn't work, the robots just enter clearing for one frame and then exit it. I think it might have something to do with the behavior changing over to the closer robot and somehow resetting the go_clear flag or something.